lesson_04_homework.rb

```diff
@@ -1,37 +1,62 @@
 # Code Reading
 
 def fizzbuzz(num)
-  # Example comment line
+  # Defining  the fizzbuzz method with num  value
 
   case
+    # statement to match  the  condition  with  the  value
     when num % 15 == 0 then "FizzBuzz"
+      # Condition : if division  of num value  with 15 = 0 then  print "fizzbuzz"
     when num % 3  == 0 then "Fizz"
+      # condition : elsif  division of num value  with 3 = 0 then  print "Fizz"
     when num % 5  == 0 then "Buzz"
+      # condition : elsif division of num value with 5 = 0 then print "Buzz"
     else num
+      # else  print  num
   end
+  #CLose the statement
 end
+# close the method
 
 def fizz_buzz_to(limit)
+  #defining fizz_buzz_to method  with limit value
   1.upto(limit).each do |num|
+  	# iterate through the upto array  and execute the code block for each element in the array
     puts fizzbuzz(num)
+    # prints fizzbuz(num) 
   end
+  # close  the  looping  through  the  array
 end
+#close  the method
 
 # Fix Broken Code
 
-def area_of_triangle(b, h)
-  puts b x height / 2
+def area_of_triangle(base, height)
+  base * height / 2
 end
 
-base = 7
-height = 6
-
-puts area_of_triangle(height, height)
+  puts area_of_triangle(7, 6)
 
 # Coding
 
 class Waitlist
-  # add your code here
+  attr_accessor :parties
+  def initialize(parties)
+    @parties = parties
+  end 
+  
+  def list
+    parties
+  end
+
+  def add_party(i)
+    parties << i
+    parties.last
+  end
+
+  def seat
+   "Hello #{parties[0]}, this is  your seat"
```

Comment:
@joni88 in this case, you are just displaying the first element in the array and not removing it as well.